### ISSUE TITLE
Warning and http://127.0.0.1:7000/static/

### ISSUE THREAD

**Issue by davin-win-kyi** [Dingry/BunnyVisionPro#9]
Hi Everyone!

Currently I am trying to run through the BunnyVisionPro documentation, and I'm currently stuck at the step where I am trying to run the robot node, and am currently getting the following issues (along with nothing showing up at http://127.0.0.1:7000/static/):

```
(base) davin123@davin123-MS-7C56:~$ docker exec -it bunny_teleop_server bash
root@docker-desktop:/server# run_robot_server -c /server/bunny_teleop_server/configs/camera_config/visionpro_config.yml -k /server/bunny_teleop_server/configs/kinematics_config/bimanual_xarm7_ability.yml -comm /server/bunny_teleop_server/configs/communication_config/sim_web_visualizer.yml
/usr/local/lib/python3.10/dist-packages/sim_web_visualizer/__init__.py:15: UserWarning: No module named 'sapien'
  warnings.warn(str(e))
/usr/local/lib/python3.10/dist-packages/sim_web_visualizer/__init__.py:16: UserWarning: 
No Sapien python library installed. Disable Sapien Visualizer.
 If you want to Sapien Visualizer, please consider install it via: pip3 install sapien
  warnings.warn(
/usr/local/lib/python3.10/dist-packages/sim_web_visualizer/__init__.py:25: UserWarning: No module named 'isaacgym'
  warnings.warn(str(e))
/usr/local/lib/python3.10/dist-packages/sim_web_visualizer/__init__.py:26: UserWarning: 
No isaacgym python library installed. Disable IsaacGym Visualizer.
If you want to IsaacGym Visualizer, please consider install it via the following URL: https://developer.nvidia.com/isaac-gym
  warnings.warn(
You can open the visualizer by visiting the following URL:
http://127.0.0.1:7000/static/
Deleting all previous scene asset.
/usr/local/lib/python3.10/dist-packages/numpy/lib/twodim_base.py:295: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  v = asanyarray(v)
Teleop ZMQ Server: Waiting for connection to tcp://*:5500
Teleop Server: Starting new handling cycle for initialization config.
Teleop ZMQ Server: Waiting for initialization config from teleop client.
```
I was wondering if we are expected to download isaacgym or sapien ourselves? Has anyone else ran into this error or does anyone know what might be going on?

Thank you, 
Davin Win Kyi

**Comment by Dingry** [comment]
Hi, this warning can be safely disregarded. Our current setup uses the base visualizer, which operates independently of both SAPIEN and the IssacGym visualizer. Therefore, this particular warning doesn't affect the functionality of our system.

**Comment by davin-win-kyi** [comment]
I see, when I try to visit http://127.0.0.1:7000/static/ non of the visualizations that show up in the documentation show up for me. I’m assuming that Isaac gym and sapien were used in the previous version, but that now it uses the base visualizer. But none of the visualizations are showing up. I followed all the steps up to the one I mentioned above, is there something that I might be missing? 


Thank you ahead of time! 
-Davin Win Kyi.

**Comment by davin-win-kyi** [comment]
Hi Everyone! I was able to resolve this issue! Instead of having: 
`docker run -it --net host --name bunny_teleop_server yzqin/bunny_teleop_server bash`

It worked for me when I did:
`docker run -it --name bunny_teleop_server -p 7000:7000 yzqin/bunny_teleop_server bash`

In my situation, specifying the port 7000:7000 helped. With the use of host --name I assume that it did not work since the network stack from the docker container and the host will both use 7000. But, if you specify 7000:7000, it means that it will map between port 7000 of the container and port 7000 of the host. This prevents the host and the docker container from using the same PORT 7000. 

In addition, I would like to add that when running the client, run it inside of the docker container. 

Hope this helps!
-Davin Win Kyi.
